src/mame/vsystem/f1gp.cpp

```diff
@@ -826,9 +840,19 @@ void f1gp_state::f1gp(machine_config &config)
 
 	ACIA6850(config, m_acia, 0);
 	m_acia->irq_handler().set_inputline("sub", M68K_IRQ_3);
-	m_acia->txd_handler().set("acia", FUNC(acia6850_device::write_rxd)); // loopback for now
+	m_acia->txd_handler().set("rs232a", FUNC(rs232_port_device::write_txd));
 
-	clock_device &acia_clock(CLOCK(config, "acia_clock", 1'000'000)); // guessed
+	// dual DE-9 ports
+	// COM-IN (inner) and COM-OUT (outer) according to manual
+	rs232_port_device &rs232a(RS232_PORT(config, "rs232a", default_rs232_devices, "loopback"));
+	rs232a.rxd_handler().set("rs232b", FUNC(rs232_port_device::write_txd));
```

Comment:
Instead of `rs232a` and `rs232b` a better name might be `com_in` and `com_out` then? I assume `rs232a` would be the COM-OUT, why does it have an `rxd` handler? Isn't it only used to transmit data?